source/renderer/app/components/staking/delegation-center/helpers.js

```diff
@@ -22,10 +22,7 @@ export const generateFieldPanel = (labels: any, values: any, index: number) => {
   const labelStr = labels[index];
   const valueStr = value.toString();
   let zeroValues = '';
-  if (
-    (index === 1 && valueStr.length < values[index + 1].toString().length) ||
-    (index === 0 && valueStr.length < EPOCH_MAX_LENGTH)
-  ) {
+  if (index === 1 && valueStr.length < values[index + 1].toString().length) {
```

Comment:
Are we sure that the condition `index === 0 && valueStr.length < EPOCH_MAX_LENGTH` is not needed anymore?

Reply:
@mchappell  I agree with D, we need to check that